README.md

```diff
@@ -185,13 +185,24 @@ The Console Oomph Installer settings (see also [Eclipse runtime options](https:/
   -Doomph.redirection.setups="index:/->$PWD/my-custom-setups/"
   ```
 
-* `oomph.redirection.*` - location of additional setup model (also contain org.eclipse.setup file), uses for override basic setup model location, ex.:
+* `oomph.redirection.*` overrides - location of additional setup model (also contain org.eclipse.setup file), uses for override basic setup model location, ex.:
 
   ```bash
   # Basic setup model location
   -Doomph.redirection.setups="index:/->http://git.eclipse.org/c/oomph/org.eclipse.oomph.git/plain/setups/"
   # Additional setup model location (overrides product and project catalog)
-  -Doomph.redirection.setupsDir="index:/->$PWD/my-custom-setups/"
+  -Doomph.redirection.setupsDir="index:/redirectable.projects.setup->$PWD/my-custom-setups/"
```

Comment:
It should be without a redirection name, because used destination directory - "index:/->$PWD/my-custom-setups/".